app/views/external_works/_blurb.html.erb

```diff
@@ -14,6 +14,11 @@
       <% fandoms = external_work.tag_groups["Fandom"] %>
       <%= fandoms.collect{ |tag| link_to_tag_works(tag) }.join(", ").html_safe if fandoms %>
     </h5>
+    <% unless external_work.language.blank? %>
```

Comment:
Just a drive-by comment: this should probably be at the start of the stats `dl`, since that's where language is on work blurbs.

Reply:
The best I come up with will look like this. Does it look fine, or should language be in a separate row from the rest of stats?

![Screenshot 2022-08-15 at 17-57-13 (development) Show External Work Example Archive](https://user-images.githubusercontent.com/40988246/184610395-32453852-be1f-4b1f-ad17-370bcf405a83.png)

Its ERB code looks like this:
```
  <!--stats-->
  <dl class="stats">
    <% unless external_work.language.blank? %>
      <dt class="language"><%= ts("Language:") %></dt>
      <dd class="language"><%= external_work.language.name %></dd>
    <% end %>
    <% if Bookmark.count_visible_bookmarks(external_work) > 0 %>
    <dt><%= ts("Bookmarks") + ": " %></dt>
    <dd><%= link_to_bookmarkable_bookmarks(external_work) %></dd>
    <% end %>
```

On second thought, if we are to add language into the external work blurb, we might want to add it to the bookmarks view, too. What do you think?

In fact, I've just realized that the task description doesn't seems to ask to show language anywhere outside of the edit screen. I think it's something helpful to have, but I'm fine with dropping this part, too, if it's unnecessary.

Reply:
That's perfect! If you could add it to the bookmark version as well, I think that would make sense.